### PR TITLE
Created Meson conf for 8.37.

### PR DESCRIPTION
**Pull request by jpakkane**

### CODE REVIEW

**Comment by igor-raits** [comment]
- [x] if new project, master has tagged commit as only commit
- [x] if new branch, it is branched from master
- [x] contains a readme.txt
- [x] contains an upstream.wrap file
- [x] download link points to authoritative upstream location
- [x] wrap repository contains only build system files
- [x] merge request is pointed to correct target branch (not master)
- [ ] wrap works
- [x] repo does not have useless top level directory (i.e. libfoobar-1.0.0)

**Comment by igor-raits** [comment]
```
Warning: Unknown keyword arguments in target chartables: include_directories
```

**Comment by igor-raits** [comment]
```
Checking for function "strerror": NO
```

It's impossible, because I have glibc where this function comes

**Comment by jpakkane** [comment]
Updated.

**Comment by fzwoch** [comment]
Not sure why this has not been merged yet. But at least as of writing the URL does not have this (slightly outdated) library version anymore.

Changing
```
source_url = ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre-8.37.tar.bz2
```
to
```
source_url = https://ftp.pcre.org/pub/pcre/pcre-8.37.tar.bz2
```
seems to handle the download at least.

I could not test it though as these raw files are different than the final wrapdb result(?)

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+pcretest = executable('pcretest',
+  'pcretest.c', 'pcre_printint.c',
+  c_args : '-DHAVE_CONFIG_H',
+  link_with : [pcre8, pcre_posix],
+  include_directories : conf_inc,
+)
+# Running test not added yet because it is
+# complicated.
+
+pcre_dep = declare_dependency(libraries : pcre8,
```
Comment (fzwoch) [inline meson.build:74]: I had to change `libraries` here to `link_with` to make it work.

**Comment by sarum9in** [comment]
Please rebase or open new one @fzwoch .

**Pull request closed without merging**